src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/MockSpringTest.java

```diff
@@ -32,14 +40,46 @@
 import uk.gov.service.notify.NotificationClient;
 
 import java.io.UnsupportedEncodingException;
+import java.time.Duration;
 import java.util.List;
 
 @RunWith(SpringRunner.class)
 @SpringBootTest
+@DirtiesContext
 @AutoConfigureMockMvc
 @TestPropertySource("/environment.properties")
+@ContextConfiguration(initializers = {MockSpringTest.Initializer.class})
 public abstract class MockSpringTest {
 
+    public static class Initializer implements ApplicationContextInitializer<ConfigurableApplicationContext> {
+        public void initialize(@NotNull ConfigurableApplicationContext configurableApplicationContext) {
+            TestPropertyValues.of(
+                "spring.datasource.claimstore.url=" + claimStorePostgreSQLContainer.getJdbcUrl(),
+                "spring.datasource.claimstore.username=" + claimStorePostgreSQLContainer.getUsername(),
+                "spring.datasource.claimstore.password=" + claimStorePostgreSQLContainer.getPassword(),
+                "spring.datasource.scheduler.url=" + schedulerPostgreSQLContainer.getJdbcUrl(),
+                "spring.datasource.scheduler.username=" + schedulerPostgreSQLContainer.getUsername(),
+                "spring.datasource.scheduler.password=" + schedulerPostgreSQLContainer.getPassword()
+            ).applyTo(configurableApplicationContext.getEnvironment());
+        }
+    }
+
+    @ClassRule
```

Comment:
Does it impact time needed to run IT? I guess new container will be started for each test class.